autoendpoint/src/extractors/subscription.rs

```diff
@@ -196,13 +196,21 @@ fn version_1_validation(token: &[u8]) -> ApiResult<()> {
 /// in standard base64 encoding. (Both of these violate the VAPID RFC)
 /// Prior python versions ignored these errors, so we should too.
 fn decode_public_key(public_key: &str) -> ApiResult<Vec<u8>> {
-    let encoding = if public_key.contains(['/', '+']) {
-        base64::STANDARD_NO_PAD
+    let engine = if public_key.contains(['/', '+']) {
+        base64::engine::fast_portable::FastPortable::from(
```

Comment:
Let's take advantage of `FastPortable::from` being a const method and make all them constants (at least outside of tests, also have some in `endpoint.rs` and `notification.rs`), its [docs recommend it](https://docs.rs/base64/latest/base64/engine/fast_portable/struct.FastPortable.html#method.from) even though they're not very expensive to construct.

Reply:
Ah, cool! Didn't see that in the docs. I was mostly just focused on getting the stuff working.